synapse/api/room_versions.py

```diff
@@ -82,12 +82,12 @@ class RoomVersions(object):
         StateResolutionVersions.V2,
         enforce_key_validity=False,
     )
-    VDH_TEST_KEY_VALIDITY = RoomVersion(
-        "vdh-test-key-validity",
-        RoomDisposition.UNSTABLE,
+    V5 = RoomVersion(
+        "5",
+        RoomDisposition.STABLE,
         EventFormatVersions.V3,
         StateResolutionVersions.V2,
-        enforce_key_validity=False,
+        enforce_key_validity=True,
```

Comment:
this was a fail in #5348: https://github.com/matrix-org/synapse/pull/5348/files#r290665308